examples/metadata_demo.ipynb

```diff
@@ -139,95 +152,246 @@
     "for model in models:\n",
     "    body = V1Model(**model)\n",
     "    try:\n",
-    "        # Create a Model Metadata entry.\n",
     "        api_response = api_instance.model_metadata_service_create_model_metadata(body)\n",
-    "        pprint(api_response)\n",
     "    except ApiException as e:\n",
-    "        print(\n",
-    "            \"Exception when calling ModelMetadataServiceApi->model_metadata_service_create_model_metadata: %s\\n\"\n",
-    "            % e\n",
-    "        )"
+    "        print(f\"Couldn't create model: {json.loads(e.body)['message']}\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## List all model metadata entries"
+    "## List all models in the Model Catalogue"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
do we want the outputs to be part of the notebook? Maybe we do I am not sure, before commiting I was cleaning up the outputs with `jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace *.ipynb`
I guess they can be helpful to see what an output looks like

Reply:
It's nice in the sense that when you view it on github you see more or less how it should work. 
I usually before commiting restart the kernel and launch it top to bottom.